src/lib/ns/user/user.c

```diff
@@ -39,11 +39,32 @@
 #include "lib/config_parser.h"
 #include "lib/privilege.h"
 
-static int enabled = -1;
-
 int singularity_ns_user_enabled(void) {
-    singularity_message(DEBUG, "Checking user namespace enabled: %d\n", enabled);
-    return(enabled);
+    singularity_message(DEBUG, "Checking if user namespace is enabled.\n");
```

Comment:
Yeah - we need something like `singularity_ns_user_plan_on_invoking_at_some_point`.

As I said, I think there is some cleanup to doing in the userns code - I was just trying to avoid blocking this ticket on the cleanup.